source/webapp/sites/all/modules/custom/checkbook_project/customclasses/util/NodeSummaryUtil.php

```diff
@@ -70,7 +70,7 @@ static function getInitNodeSummaryTitle($nid = null){
         $title = NULL;
             if($smnid){
                 if(isset($node->widgetConfig->summaryView->templateTitleEval)){
-                    $title = eval($node->widgetConfig->summaryView->templateTitleEval);

```

Comment:
I don't think removing all these evals is good idea, these views have executable php code